src/main/java/com/ebay/myriad/scheduler/TaskFactory.java

```diff
@@ -63,6 +58,31 @@ private static String getFileName(String uri) {
             }
         }
 
+        private CommandInfo getCommandInfo() {
+            MyriadExecutorConfiguration myriadExecutorConfiguration = cfg.getMyriadExecutorConfiguration();
+            String cmdPrefix = "export CAPSULE_CACHE_DIR=`pwd`;echo $CAPSULE_CACHE_DIR; java -Dcapsule.log=verbose -jar ";
+            CommandInfo.Builder commandInfo = CommandInfo.newBuilder();
+            if (myriadExecutorConfiguration.getNodeManagerUri().isPresent() &&
+                    myriadExecutorConfiguration.getCommand().isPresent() &&
+                    myriadExecutorConfiguration.isRemoteDistribution()) {
+                String nmURI = myriadExecutorConfiguration.getNodeManagerUri().get();
+                URI executorURI = URI.newBuilder().setValue(nmURI).setExtract(true).build();
+                //@Todo: determine if "root" or "" is the proper default, to keep backward compatiblility
+                //keeping root for now
```

Comment:
I think we can defer the user cleanup to #47 and leave this PR for just the binary distribution. In that regard, you can remove the TODOs and we'll address them for #47